proxytest/proxytest.go

```diff
@@ -58,6 +58,7 @@ type HostEmulator interface {
 	CallOnRequestBody(contextID uint32, body []byte, endOfStream bool) types.Action
 	CompleteHttpContext(contextID uint32)
 	GetCurrentHttpStreamAction(contextID uint32) types.Action
+	GetHttpRequestHeaders(contextID uint32) types.Headers
```

Comment:
please rename to `GetRequestHeaders` to be better aligned.

Reply:
Can you provide your option?

Reply:
I mean make this aligned with `GetCurrentHttpStreamAction` right above which has same functionality for streamAction.